scripts/distribute-version.py

```diff
@@ -0,0 +1,14 @@
+import subprocess
+import time
+
+
+def push_pod(name):
```

Comment:
This job works only for GA versions, right?

Reply:
yes, it works only for /^trigger-distribute-version-.*/ branches